press/press/doctype/app_source/app_source.py

```diff
@@ -61,6 +61,9 @@ def validate_duplicate_versions(self):
 
 	def before_save(self):
 		# Assumes repository_url looks like https://github.com/frappe/erpnext
+		if self.repository_url.endswith(".git"):
+			self.repository_url = self.repository_url[:-4]
```

Comment:
```suggestion
		self.repository_url = self.repository_url.removesuffix(".git")
```

Reply:
"batteries included" is an understatement